sources/lib/Resources/config/services/pomm.yml

```diff
@@ -8,10 +8,16 @@ services:
         arguments: ['@pomm']
         tags:
             - { name: request.param_converter, converter: pomm.entity }
+
     pomm.serializer.normalizer:
         class: 'PommProject\SymfonyBridge\Serializer\Normalizer\FlexibleEntityNormalizer'
         tags:
-            - { name: serializer.normalizer }
+            - { name: serializer.normalizer, priority: 999 }
```

Comment:
Hmm, but why? I mean what is the context where this normalizer will be used? Is it needed to declare the `serializer.normalizer` tag in this bundle (maybe leave this to the end user)?